### DISCUSSION TITLE
Move nvchad to it's own folder

### DISCUSSION THREAD

**Discussion by uzuki-P** [NvChad/NvChad#2804] [Q&A] [answered]
Is there a way to move the nvchad template folder to `~/.config/nvim/lua/nvchad`. I'm trying to make my own config while want to easy switch to nvchad. I got stuck because [this line](https://github.com/NvChad/NvChad/blob/v2.5/lua/nvconfig.lua#L86) on "chadrc".

**Comment by siduck** [comment, marked as answer]
clone nvchad at `~/.config/nvchad` and add the alias

```bash
alias nv='NVIM_APPNAME="nvchad" nvim'
```

and this `nv` will run nvchad config only!!
